type_check_failed,error_msgs
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in get: Name 'x' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 25, in get_dstring: No attribute 'reader' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in parse_expr7: Name 'pos' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in p_top_type_symbol: Name 'tdfn' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in p_top_mixin_callatom_after_callatom: Name 'd' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in p_top_interpret_symbol_arrow_lcb_symbol_dots_symbol_rcb: Name 'thing' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in solver_name: Name 'name' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in sorts: Name 'width' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in uninterpretedsort: Name 's' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in exist_quant: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in forward_image: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in lit_id: Name 'key' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in handle: No attribute 'start' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 54, in monitor: No attribute '__iter__' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in function_model_to_clauses: Name 'c' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in substitute: unsupported operand type(s) for item assignment: int [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in declare_symbol: No attribute 'append' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in emit_assign_simple: No attribute 'args' on List[nothing] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in emit_sequence: Name 'a' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_late_contributions: No attribute 'check_response' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_latest: Name 'latest' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in days_since_release: Name 'release_date' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in avgmapper: No attribute 'next' on _csv._reader [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in mapper: Name 'word' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in main: Name 'counts' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in encode_queryset: Name 'obj' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in filter_expired_cookies: Name 'kvp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in build_grad: Name 'left_increment' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in shared_iobuf_size: Name 'num_dev' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_contributions_to_candidate: Name 'contributions' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_detail_using_id: No attribute 'check_response' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in parse_response: unsupported operand type(s) for item retrieval: 'types: module' and 'Any' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in _parse_args: Function OptionParser.parse_args was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in parseNumbers: Name 'm3' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_form_types: No attribute 'check_response' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_superpacs: No attribute 'check_response' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_latest: Name 'latest' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_leaders: No attribute 'check_response' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in _pretty_print: unsupported operand type(s) for item retrieval: Dict[str, str] and int [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _get_default_ordering: No attribute 'model_admin' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get: No attribute 'get_queryset' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in splitpath: Name 'tail' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in deploy_api_gateway: Name 'endpoint_url' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_create_lambda_function: No attribute 'update_lambda_function' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_rollback_lambda_function_version: Name 'too_many_versions' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_deploy_api_gateway: Name 'z' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in test_b64_pattern: Function base64.b64encode was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: 'tuple' object is not callable [not-callable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in compress: No attribute 'cleaned_data' on dict [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in sort_retention_periods: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in users: Name 'start' is not defined [name-error]
File ""dummpy_input_file"", line 9, in users: 'time' object is not callable [not-callable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_terminal: Name 'terminal' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __str__: Name 'ss' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in has_scope: No attribute 'get_scope_names_set' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in cube_coordinates_all: Name 'x' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in delete: No attribute 'status_code' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in getEmailAdresses: unsupported operand type(s) for item retrieval: List[nothing] and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in sendMessage: No attribute 'json' on Dict[str, str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in markAsRead: No attribute 'auth' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in getCalendars: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in VariableDef: Function unsolveable.VariableDef expects 1 arg(s), got 7 [wrong-arg-count]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 28, in Parameter: No attribute 'upper' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_terminal: Name 'terminals' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _configure_merge: Name 'out_shapes' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in post: No attribute 'get_object' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 27, in testAnonymousUser: No attribute 'get_anonymous_id' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 36, in testEnabledPromotedAndDisabledExperiment: No attribute 'assertNotEqual' on None [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in _id_from_session: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in _id_from_user: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in experiment: Name 'group_name' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in get_grad_back: Name 'right_increment' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_grad_asnumpyarray: No attribute 'dtype' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 3)"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_wsgi_event: Name 'request' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_handler: No attribute 'handler' on Dict[str, Union[Dict[str, str], str]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_cli_utility: No attribute 'print_logs' on List[Dict[str, str]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 36, in testConversionReportGenerator: No attribute 'save' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in gammln: unsupported operand type(s) for +: List[float] and float [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in load_data: Name 'phase' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in CalculateCellPadding: Name 'penFactor' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in CalculateBounds: Name 'bounds' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in GetCombinedLists: Can't assign attribute 'innerCellWidth' on int [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in GetValue: Name 's' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in GetValue: Name 's' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in GetValue: Name 's' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: No attribute 'SetValue' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in GetSubstitutionInfo: Name 'info' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in resizeLabel: No attribute 'grdSummary' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __call__: Name 'cols' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in GetCellWidths: Name 'numNonEmptyTexts' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in Print: No attribute 'left' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in GetTexts: No attribute 'lv' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in GetTextColor: Name 'attr' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in GetTexts: No attribute 'lv' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in GetImages: No attribute 'lv' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in DrawDecoration: Name 'y' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __repr__: Name 'strs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: No attribute 'Show' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_loadIntoDataFrame: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_build_series: No attribute 'session' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_variable_relationship: Name 'var' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_source_relationship: Name 'source' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in get_method_by_description: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in get_values_by_series: Name 'series' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in calcSeason: Name 'month' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in configure: Can't assign attribute 'log_traffic' on tuple [not-writable]"
False,
True,"File ""dummpy_input_file"", line 4, in connect: No attribute 'client' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in _build: Can't assign attribute '_config' on str [not-writable]"
False,
True,"File ""dummpy_input_file"", line 20, in __init__: Built-in function setattr was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in connect: Name 'conn' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in getSmallEnvDict: Name 'smallEnvDict' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in convertTimeStamp: No attribute 'replace' on time.struct_time [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in envFileAsDictionary: Built-in function open was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in OnPaint: unsupported operand type(s) for item assignment: Tuple[int, int, int, int] [unsupported-operands]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in GetValue: Name 's' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in run_pyinstaller: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 14, in __init__: No attribute 'dispatcher' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in gen_class: Name 'lsave' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_escape_fallback_encoder: No attribute 'escape' on Type[Custom] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_skip_nextset: Name 'cur' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_largedata: Name 'cur' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _check_mysql_exception: Name 'errorclass' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in _pydate2wxdate: Name 'dmy' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in PrintPage: Name 'i' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in SubtractDecorations: Name 'fmt' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in DoPrint: Name 'printer' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in resetCB: Name 'cDb' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in colorize: No attribute 'end' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 32, in flush: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in qcl_exists: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in __init__: No attribute 'titleBar' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in initiateObjectListView: No attribute 'olv' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_sum_squared_limits: Name 'expected_result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in test_smoothL1_random: Name 'J2' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in onSetScriptTitle: Name 'scriptPane' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: Can't assign attribute 'memDB' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in stopEditSeries: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in setBoxInterval: Name 'value' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in buildPlotInfo: No attribute 'editID' on List[Tuple[str, Any]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in processTasks: Name 'i' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in getCompletedTasks: No attribute 'completedTasks' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in processTerminate: Name 'n' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __exit__: Can't assign attribute 'end' on tuple [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __init__: Can't assign attribute 'frame' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in onDeleteBtn: Name 'length' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: Name 'PrbExc' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in onKeyPressed: unsupported operand type(s) for item assignment: Sequence[str] [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in setup: Name 'engine' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 33, in add_series_bulk_data: No attribute 'begin_date_time' on list [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in add_site: Name 'spatial_ref' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in _init_toolbar: Name 'text' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in callback: Name 'ind' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in onResetFilter: Name 'recordService' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in onSave: Name 'wiz' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 28, in fetch: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in register_plugin: Name 'lineprofiler_act' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in select_file: Name 'filename' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in show_tree: Name 'col' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 22, in load_data: unsupported operand type(s) for item retrieval: 'fid: BinaryIO' and 'next_stat_line: int' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in OnInit: unsupported operand type(s) for item assignment: str [unsupported-operands]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in query_filter: Name 'item_present' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in new_goal: Name 'g' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in do_ls: Name '_' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 35, in autotune: No attribute 'record' on float [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in read_images: Can't assign attribute 'features' on str [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in initialize: Name 'll' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 26, in refresh_devices: Name 'deviceKey' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_http_method: Name 'i' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in get_url: Name 'arg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in popen: Function str.join was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in init_log: Name 'fh' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in process: Name 'u_string' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in get_translations: Name 'translations' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in buildOlv: Name 'columns' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in OnBtnOKButton: Name 'code' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in __init__: No attribute '_init_database' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in refreshConnectionInfo: No attribute '_mgr' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in test_normalize_paths: No attribute 'assertEqual' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in launch_threads: Name 'thread' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in cyfieithu_testun: Name 'translations' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: Can't assign attribute 'env' on str [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in with_feature: Name 'wrapper' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in password: Function range.__init__ was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in random_value: Function random.choice was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in DeleteVbaModule: Name 'vba_module_todel' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in generate_trigger_function: Name 'gen_vba' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in addItem: No attribute 'append' on Callable [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _getContent: Name 'filepath' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in timeout: Name 'cache' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in panos_time: Name 's' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in get_wildfire_apikey: Name 'c' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in set_firewall_apikey: Name 'apikey_entity' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 10, in delete_firewall_apikey: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in __set_stream_response: unsupported operand type(s) for item assignment: str [unsupported-operands]
File ""dummpy_input_file"", line 18, in __set_stream_response: unsupported operand type(s) for item assignment: None [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in timestamp: 'deprecated' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 27, in VariableDef: Name 'val' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in Expression: Name 'expr' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 35, in GetContext: Name 'indices' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 262, in Completions: Name 'e' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in start_analysing_stream: No attribute 'is_active' on pyaudio.PyAudio [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in __init__: Can't assign attribute '_direct_commands' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in update: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in url: No attribute '_url' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in setup_default_args: Name 'rt_grp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in depinfo_for_file: Name 'archive' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in forget: Name 'head' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in regenerate_leaf: Name 'versions' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in test_pkginfo_from_file_exc_and_handler: No attribute 'call_args' on Exception [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in refresh_devices_from_panorama: Name 'device' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 33, in syncreboot: No attribute '_logger' on float [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in element: No attribute 'destination' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in batch_end: Name 'payload' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_pooling: Name 'pool_test_args' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in deconv_summary_page: unsupported operand type(s) for item assignment: str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_uniform: No attribute 'fill' on Tuple[Any, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_download_url: Name 'rd' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_package_finder_cm: Name 'finder' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_pkginfo: Name 'pkg' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in basic_prep: Name 'req' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_handle_requirement_httperror: Name 'req' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_regenerate_index: 'pprint' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in createOpener: Name 'opener' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in panoramaCommit: Name 'panurl' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 35, in main_splunk: Name 'ip_field' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in xpath: Name 'parent_xpath' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in create: Name 'xpath' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in generate_xapi: Name 'kwargs' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 10, in __get_xpath_scope: No attribute 'is_panorama' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 10, in _retrieve_api_key: Name 'xapi' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_initial_widget_is_hidden: Name 'original_widget' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in handle_tcp: No attribute 'recv' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 22, in handle: Name 'addr' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in __str__: No attribute 'iteritems' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in to_dotfile: Name 'attribs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in xml_result: Name 'elem' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in build: Name 'cmd' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in run: No attribute 'build' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 17, in main: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 37, in get_value_from_post_data: Name 'values' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in to_dict: No attribute '_XSI_TYPE' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in add: Name 'tlo_adds' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in to_dict: Name 'd' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in from_list: Name 'mlist' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in _message: No attribute 'MESSAGE' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in to_dotfile: Name 'attribs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in _link_attribute: No attribute 'MAPPING_TO_COLLECTION_TYPE_RE' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _attribute_cell: Name 'type_descr' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in _operation_cell: No attribute 'vararg' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in _color: Name 'green' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in _trade_dates: Name 'li_even' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 15, in daily: Name 'nds' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in ctx_assert_warnings: No attribute 'assertTrue' on List[warnings.WarningMessage] [attribute-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_round_trip_full_dict: Name 'dict2' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_round_trip_full: Name 'ent2' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in setUpClass: Name 'slist' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_reset: No attribute 'assertEqual' on Tuple[int, int, int] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 10, in daily1: Name 's' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in write_pickle: No attribute 'dump' on BinaryIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in syno_populating: No attribute 'relavent_val_pos' on Dict[nothing, nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 11, in process_query_result: Can't assign attribute 'line_curr' on bool [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 36, in tq_generate_thesaurus_buffer: No attribute '__iter__' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in data_from_path: Name 'stream' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in arch_to_add_map: Name 'pkgi' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_ordinalities: No attribute 'assertTrue' on range [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_xsi_type_required: No attribute 'assertRaises' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 6)"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 29, in oneKnn: unsupported operand type(s) for item retrieval: 'int' and '-1: int' [unsupported-operands]"
False,
True,"invalid syntax (<unknown>, line 4)"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 7)"
False,
True,"File ""dummpy_input_file"", line 19, in addEvidence: No attribute 'training_data' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in dataifywine: Name 'foo' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in create_script_tagnames: Built-in function open was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_voice: Name 'response' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_no_photo: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_sms: Name 'response' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in read_config: Name 'config_password' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in init_app: Name 'args' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_empty: Name 'table' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in from_dict: Name 'klass' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in name_list: Name 'names' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 9)"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in featOption: Name 'sym' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in parse: No attribute 'build' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in __init__: Function Logger.addHandler was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in serialization: Name 'tmp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in _fix_value: Name 'new_value' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in handle_future: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in buildChildren: Name 'Title_' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in from_dict: Name 'get' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in to_dict: No attribute 'value' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in calcgamma: Name 'gamma' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in read_symbols: No attribute 'append' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 10, in update_my_data: Name 'ls_missed_syms' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 38, in analyze_transactions: unsupported operand type(s) for item retrieval: 'volume: int' and '3: int' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in getMonths: No attribute 'month' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in getNextOptionClose: Name 'friday' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in random_string: Function SystemRandom.choice was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
